platforms/documentation/docs/src/docs/userguide/releases/upgrading/upgrading_version_8.adoc

```diff
@@ -52,6 +52,10 @@ Apache SSHD has been updated from 2.0.0 to https://mina.apache.org/sshd-project/
 
 http://www.jcraft.com/jsch/[JSch] has been replaced by https://github.com/mwiede/jsch[`com.github.mwiede:jsch`] and updated from 0.1.55 to https://github.com/mwiede/jsch/releases/tag/jsch-0.2.16[0.2.16]
 
+==== Upgrade to ASM 9.6
+
+ASM was upgraded from 9.5 to https://asm.ow2.io/versions.html[9.6] for better support of multi-release jars.
```

Comment:
```suggestion
ASM was upgraded from 9.6 to https://asm.ow2.io/versions.html[9.6] for better support of multi-release jars.
```

Reply:
it's "from"?

Reply:
Nevermind, it says `from` and not to. Ignore that

Reply:
My bad :)

Reply:
NP:)